test/jdk/javax/swing/JRadioButton/bug4380543.java

```diff
@@ -46,20 +45,19 @@
 import javax.swing.UIManager;
 import javax.swing.UnsupportedLookAndFeelException;
 
-import java.util.HashMap;
-import java.util.Map;
-
 public class bug4380543 {
     static testFrame testObj;
     static String instructions
-            = "INSTRUCTIONS:" +
-            "\n 1. This is a Windows specific test. If you are not on " +
-            "Windows, press Pass." +
-            "\n 2. Check if the Left insets(margins) is set visually " +
-            "similar to other three sides around Radio Button and CheckBox" +
-            "(insets set to 20 on all 4 sides)." +
-            "\n 3. If Left insets(margins) appear Empty, press Fail, " +
-            "else press Pass.";
+            = """
+            INSTRUCTIONS:
+               1. Check if the Left insets(margins) is set visually
+                  similar to other three sides around Radio Button
+                   and CheckBox (insets set to 20 on all 4 sides).
+               2. Rendering depends on OS and its Supported Look and Feel.
+                  Verify only with those L&F where Margin is Visible.
+               3. If Left insets(margins) appear Empty, press Fail,
+                  else press Pass.
```

Comment:
```suggestion
               1. Check if the Left inset(margin) is set visually
                  similar to other three sides around Radio Button
                   and CheckBox (insets set to 20 on all 4 sides).
               2. Rendering depends on OS and supported Look and Feels.
                  Verify only with those L&F where margins are visible.
               3. If Left inset(margin) appears too small, press Fail,
                  else press Pass.
```

I think _left inset_ or _margin_ is better — it's one instance.

You may want to add an additional space to all the lines without numbers to visually align the text to the start of the text after the number. I assume this is why you have an extra space before ‘and’.

Reply:
Done.